### PR TITLE
ui issues fixed

### PR DESCRIPTION
**Pull request by adeelehsan**
VAN-601,607,608,615

### CODE REVIEW

File: src/_style.scss [hunk 1]
```diff
@@ -647,3 +647,6 @@ select.form-control {
       font-weight: 700;
     }
 }
+.email-top-domain-suggestion-alert-link {
+  color: #454545 !important;
```
Comment (waheedahmed) [inline src/_style.scss:651]: Use `$primary`.

**Review by waheedahmed — COMMENTED** [review]
Just one minor nit otherwise looks good.

**Review by waheedahmed — APPROVED** [review]

File: src/forgot-password/ForgotPasswordAlert.jsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ const ForgotPasswordAlert = (props) => {
 
   let message = '';
   let heading = intl.formatMessage(messages['forgot.password.error.alert.title']);
+  const supporUrl = getConfig().PASSWORD_RESET_SUPPORT_LINK;
```
Comment (shafqatfarhan) [inline src/forgot-password/ForgotPasswordAlert.jsx:23]: Typo: `supportUrl`

File: src/forgot-password/ForgotPasswordAlert.jsx [hunk 3]
```diff
@@ -33,7 +34,11 @@ const ForgotPasswordAlert = (props) => {
           values={{
             email: <span className="data-hj-suppress">{email}</span>,
             supportLink: (
-              <Alert.Link className="alert-link" href={getConfig().PASSWORD_RESET_SUPPORT_LINK}>
+              <Alert.Link
+                className="alert-link"
+                href={supporUrl}
```
Comment (shafqatfarhan) [inline src/forgot-password/ForgotPasswordAlert.jsx:39]: typo: `supportUrl`

**Pull request merged**